### ISSUE TITLE
【2.0 API文档内测】BatchNorm2D文档问题 

### ISSUE THREAD

**Issue by ruyijidan** [PaddlePaddle/Paddle#28819]
文档问题描述：
paddle.nn.BatchNorm2D 文档评估

API文档描述是否清晰？
中文：4分，描述清楚
英文：4分，描述清楚

参数说明是否清晰
中文：5分
英文：5分

返回/形状说明是否清晰
中文：5分
英文：5分

示例代码是否有效？
中文：3.5分
英文：3.5分

中英文内容是否一致？
大体一致

其他
英文排版有问题，后面都是无关的内容。
中英文文档都是开启静态图，在有些时候是不希望训练BN的参数的，什么时候能有affine参数来配置是否训练BN中的参数。

整体评价
4分，深度学习中最常用的操作；正常按照文档使用不会有太大问题，
